index.bs

```diff
@@ -95,7 +95,7 @@ A <dfn export>readable stream</dfn> represents a source of data, from which you
 <em>out</em> of a readable stream. Concretely, a readable stream is an instance of the {{ReadableStream}} class.
 
 Although a readable stream can be created with arbitrary behavior, most readable streams wrap a lower-level I/O source,
-called the <dfn>underlying source</dfn>. There are two types of underlying source: push sources and pull sources.
+called the <dfn>underlying source</dfn> ([[#rs-constructor|interface description]]). There are two types of underlying source: push sources and pull sources.
```

Comment:
This isn't good, because we're currently talking about the general concept of underlying sources, not the JS objects that sometimes (for web-developer-created streams) embody them. Such a link belongs later, where we say "For web developer–created streams, the implementation details of a source are provided by an object with certain methods and properties that is passed to the ReadableStream() constructor." But, that already links to the constructor, so I'm not sure if there's anything to do here.

Reply:
I see your point. The problem that I’m trying to solve here is the following flow:

1. Go to stream spec
1. Click “ReadableStream” in ToC
1. Get confused, maybe click `ReadableStream` again?
1. Okay, I’m at the class, I see `underlyingSource`, but I can’t click it. The paragraph has a clickable “underlying Source”, tho. So let’s click that.
1. I was already here...

Maybe we can turn `underlyingSource` in the class definition into a link to the note? Would that be acceptable?